### DISCUSSION TITLE
openshift-install create cluster not working on 4.13.0-0.okd-2023-09-03-082426

### DISCUSSION THREAD

**Discussion by psgandalf** [okd-project/okd#1744] [Q&A]
Hi,
When I install a cluster on vcenter with an intall-config.yaml created with an earlier version 4.12.0-0.okd-2023-02-04-212953 it creates the bootstrap and the masters. The bootstrap then should add extra ip:s for the api endpoint but that never happens. If I run the same yaml on the older version everything works. 
I cannot create the install-config.yaml in the last two versions at least (I have an other discussion about that)

**Comment by JaimeMagiera** [comment]
You'll want to provide the install-config.yaml that works, the example of the failure to create a newer install-config.yaml, and a must gather archive.

**Reply by psgandalf** [reply]
`additionalTrustBundlePolicy: Proxyonly
apiVersion: v1
baseDomain: mydomain.local
compute:
- architecture: amd64
  hyperthreading: Enabled
  name: worker
  platform: {}
  replicas: 3
controlPlane:
  architecture: amd64
  hyperthreading: Enabled
  name: master
  platform: {}
  replicas: 3
metadata:
  creationTimestamp: null
  name: okd
networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  machineNetwork:
  - cidr: 192.168.1.0/24
  networkType: OVNKubernetes
  serviceNetwork:
  - 172.30.0.0/16
platform:
  vsphere:
    apiVIPs:
    - 192.168.1.165
    cluster: R730 Cluster
    datacenter: Datacenter
    defaultDatastore: R5
    ingressVIPs:
    - 192.168.1.166
    network: VM Network
    password: mypassw0rd
    username: administrator@mydomain.local
    vCenter: vcenter.mydomain.local
publish: External
pullSecret: '{"auths":{"cloud.openshift.com":"longerirl"}}'
sshKey: 'ssh-key123 notanactualkey'`

Truncated and changed domains.
The latest version of installer that works is openshift-install-linux-4.13.0-0.okd-2023-08-04. The later releases never get an api endpoint.

**Comment by psgandalf** [comment]
https://1drv.ms/u/s!ArBx0wVPqvX8g8g_dVm3avRG2nNx8A?e=SnGXlj
Link to must-gather archive

**Reply by vrutkovs** [reply]
Since the installation never completed, you should upload the [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

**Reply by psgandalf** [reply]
I did not save the logs from the failed installation. Must-gather is from the running cluster installed with the 23-08-04 version.
